### DISCUSSION TITLE
Is it possible to create svelte template?

### DISCUSSION THREAD

**Discussion by paolotiu** [codesandbox/sandpack#167] [Q&A] [answered]
I tried doing a custom setup following https://codesandbox.io/api/v1/sandboxes/svelte/sandpack, but it doesn't seem to work. How would I go about this? Thank you!

After looking into this more, it seems that any release after v0.1.3 doesn't work with svelte. I'm using this sandbox: https://codesandbox.io/s/sandpack-test-svelte-c1by1?file=/src/App.js.

**Comment by DeMoorJasper** [comment]
It is possible to use svelte with sandpack and your sandbox works for me.

Did notice some performance issues with sandpack-svelte from using your sandbox that I'm investigating right now.

**Reply by paolotiu** [reply]
Oh sorry, the sandbox I gave was the working one using v0.0.5. Here's the one with the latest version (v.0.7.0)
https://codesandbox.io/s/sandpack-test-svelte-forked-zllvp.

**Comment by paolotiu** [comment, marked as answer]
Fixed in v0.7.1
